src/main/scala/thurloe/webservice/SwaggerConfigHttpService.scala

```diff
@@ -0,0 +1,71 @@
+package thurloe.webservice
+
+import com.gettyimages.spray.swagger.SwaggerHttpService
+import com.typesafe.config.Config
+import com.wordnik.swagger.model.ApiInfo
+import spray.http.StatusCodes
+import spray.routing.HttpService
+
+// TODO: This swagger boilerplate, and the Http.Bind in CromwellServer, should be in a dsde-common
+trait SwaggerConfigHttpService extends SwaggerHttpService with SwaggerUiHttpService {
+  def swaggerConfig: Config
+
+  override lazy val baseUrl: String = swaggerConfig.getString("baseUrl")
+  // NOTE: These two values in the conf must be kept in sync with changes to build.sbt
+  // Validated by SwaggerConfigHttpServiceSpec
+  override lazy val apiVersion: String = swaggerConfig.getString("api.version")
+  override lazy val swaggerUiVersion: String = swaggerConfig.getString("ui.version")
+
+  override lazy val docsPath: String = optString("docsPath", super.docsPath)
+  override lazy val swaggerVersion: String = optString("swaggerVersion", super.swaggerVersion)
+  override lazy val uiPath: String = optString("ui.path", super.uiPath)
+
+  override lazy val apiInfo: Option[ApiInfo] =
```

Comment:
Several of the above properties will be marked as not required also once `spray-swagger` is removed. All of `spray-swagger` has been superseded by the use of static yaml instead.